gno.land/pkg/gnoland/app.go

```diff
@@ -159,7 +159,7 @@ func InitChainer(baseApp *sdk.BaseApp, acctKpr auth.AccountKeeperI, bankKpr bank
 
 				// NOTE: comment out to ignore.
 				if !skipFailingGenesisTxs {
-					panic(res.Error)
+					panic(res.Log)
```

Comment:
slipping in by accident as most errors here, if we just get res.Error, tend to be `internal error`. if we feel like we need to discuss it, happy to do another PR.